src/Ucommerce.Sitecore.Web/Css/Sitecore8/UI.css

```diff
@@ -123,9 +123,14 @@ body * {
   height: 100%;
 }
 
+.imagePickerContainer .imagePickerContainerBody.propertyItem {
+  display: flex !important;
+}
+
 .imagePicker .imagePickerContainer.propertyPane {
   height: 335px !important;
   overflow-x: auto;
+  display: flex;

```

Comment:
Please verify that the fix doesn't break other dialogs (e.g. Content picker, Email picker)

Reply:
Already verified this with David :)